src/js/chatWindow.js

```diff
@@ -1,8 +1,20 @@
-export const addMessageToChatHistory = (m) => {
+const guardOptions = (options) => {
```

Comment:
Do you plan to write tests around this guard? I'm not sure how valuable they'll be but just curious.